components/Header/index.js

```diff
@@ -55,7 +55,7 @@ class Header extends Component {
                 height={height}
                 className="d-flex align-items-center"
               >
-                <Logo src="/static/images/logo.svg" alt="Masifunde Logo" />
+                <Logo src="/static/images/logo.svg" alt="" />
```

Comment:
This should say `Masifunde Home`, as it's a link to go to the home page.

Reply:
I am finding different examples:

Reply:
I will follow the example from the website: https://www.w3.org/WAI/tutorials/images/functional/.